app/src/org/commcare/android/framework/BreadcrumbBarFragment.java

```diff
@@ -235,6 +235,13 @@ private View findAndLoadCaseTile(final Activity activity) {
 
         final ImageButton infoButton = ((ImageButton)holder.findViewById(R.id.com_tile_holder_btn_open));
 
+        Detail contextDetail = CommCareApplication._().getCurrentSessionWrapper().getSession().
+                getDetail(inlineDetail);
+
+        if(contextDetail == null) {

```

Comment:
Oh, right! Good point, this totally doens't seem right. Will revise

On Wed, Sep 9, 2015 at 3:01 PM, Jenny Schweers <notifications@github.com>
wrote:

> In app/src/org/commcare/android/framework/BreadcrumbBarFragment.java
> <https://github.com/dimagi/commcare-odk/pull/613#discussion_r39082445>:
>
> > @@ -235,6 +235,13 @@ private View findAndLoadCaseTile(final Activity activity) {
> >
> >          final ImageButton infoButton = ((ImageButton)holder.findViewById(R.id.com_tile_holder_btn_open));
> >
> > +        Detail contextDetail = CommCareApplication._().getCurrentSessionWrapper().getSession().
> > +                getDetail(inlineDetail);
> > +
> > +        if(contextDetail == null) {
>
> wouldn't checking inlineDetail == null do the same thing?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-odk/pull/613/files#r39082445>.
>